WalletWasabi.Tests/Helpers/MemPoolInfoGenerator.cs

```diff
@@ -0,0 +1,60 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin;
+
+namespace WalletWasabi.Tests.Helpers
+{
+	public class MemPoolInfoGenerator
+	{
+		private static Random rnd = new Random();
+
+		private static int[] FeeLimits = new[]
```

Comment:
I would add `readonly` if one is not supposed to change the array.

Reply:
Done.